logs/README.md

```diff
@@ -28,11 +28,19 @@
 Returns all logs according to the provided queries and path/named-route param.
 
 - **Param**  
-  None
-- **Query**  
-  type=<LOG_TYPE>
-- **Body**  
-  None
+  type:
+
+  - `CLOUDFLARE_CACHE_PURGED`
+  - `PROFILE_DIFF_APPROVED`
+  - `PROFILE_DIFF_REJECTED`
+  - `extensionRequests`
+  - `task`
+  - `archiveDetails`
+
+- **Query**
+- Optional: `userId=[string]` for type = `archiveDetails`
+- type = <LOG_TYPE>

```

Comment:
And what is `type = <LOG_TYPE>` here?

Reply:
It is coming from previous commit, so I keep it the same